docs/dm/zh_CN/features/relay-log.md

```diff
@@ -0,0 +1,57 @@
+relay-log
+===
+
+### 索引
+- [功能介绍](#功能介绍)
+- [本地存储目录结构](#本地存储目录结构)
+- [初始同步规则](#初始同步规则)
+- [数据清理](#数据清理)
+
+
+### 功能介绍
+
+DM-worker 启动后会自动同步上游的 binlog 到配置的本地目录（使用 `DM-Ansible` 部署的默认同步目录为 `<deploy_dir>/relay_log`）。DM-worker 运行过程中会实时同步上游的 binlog 更新到本地文件，syncer 组件会实时读取本地的 relay-log 更新并同步更新至下游数据库。
+
+
+### 本地存储目录结构
+
+以下是一个 relay-log 本地存储的目录结构示例：
+
+```
+<deploy_dir>/relay_log/
+|-- 7e427cc0-091c-11e9-9e45-72b7c59d52d7.000001
+|   |-- mysql-bin.000001
+|   |-- mysql-bin.000002
+|   |-- mysql-bin.000003
+|   |-- mysql-bin.000004
+|   `-- relay.meta
+|-- 842965eb-091c-11e9-9e45-9a3bff03fa39.000002
+|   |-- mysql-bin.000001
+|   `-- relay.meta
+`-- server-uuid.index
+```
+
+- subdir 目录：DM-worker 从上游数据库同步的 binlog 会保存在同一个目录，每一个目录叫做一个 subdir，subdir 的命名格式是 <上游数据库 uuid>.<本地 subdir 序号>。当上游进行主从切换后，DM-worker 会生成一个新的 subdir 目录，并且新目录序号递增。
+- server-uuid.index： 记录了当前的 subdir 列表信息
+- relay.meta：在每个 subdir 内，用于保存已同步上游 binlog 的位置信息。
+
+
+### 初始同步规则
+
+DM-worker 每次启动（或 relay-log 从暂停状态恢复同步），从上游 binlog 哪个位置开始同步有以下几种情况：
+
+* 本地有有效的 relay_log（有效指有正确的 server-uuid.index 文件、subdir、relay.meta 文件和 binlog 文件）：会根据 relay.meta 记录的 binlog 点继续同步
+* 本地没有有效 relay_log，并且没有在 DM-worker 配置文件中指定 relay-binlog-name 或 relay-binlog-gtid：从上游最旧的 binlog 开始同步，依次同步上游所有 binlog 文件至最新
```

Comment:
if not enable GTID, upstream can do auto_position correctly. but if enable GTID, this may fail because old binlog purged. more description?